libs/experimental/langchain_experimental/agents/agent_toolkits/pandas/base.py

```diff
@@ -162,6 +162,7 @@ def create_pandas_dataframe_agent(
     include_df_in_prompt: Optional[bool] = True,
     number_of_head_rows: int = 5,
     extra_tools: Sequence[BaseTool] = (),
+    use_modin: bool = False,
```

Comment:
i think there are a number of drop in replacements for pandas  -- a Literal['pandas', 'modin'] would work better

Reply:
@eyurtsev Done